code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -9,6 +9,7 @@
  * Glass sheets
  */
 GLOBAL_LIST_INIT(glass_recipes, list ( \
+	new/datum/stack_recipe("glass shard", /obj/item/shard, time = 0, on_floor = TRUE, window_checks = TRUE), \
```

Comment:
Why are you doing window checks and putting it on floor for a glass shard

Reply:
this tbh

Reply:
Don't copy and paste kids